src/main/java/lv/ctco/javaschool/eventorganaizer/boundary/EventOrganizationApi.java

```diff
@@ -201,4 +208,25 @@ private void registerNewUserPoll(User currentUser, Answer answer) {
         pollStore.persistUserPoll(newUserPoll);
         answer.setCounter(answer.getCounter() + 1);
     }
+
+
+    @POST
+    @RolesAllowed({"ADMIN", "USER"})
+    @Path("/{id}/saveFeedback")
+    public void saveFeedback(String feedbackText, @PathParam("id") Long id) {
+        Feedback feedback = new Feedback();
+        feedback.setEvent(eventStore.getEventById(id).get());
+        feedback.setFeedbackAuthor(userStore.getCurrentUser().getUsername());
+        feedbackText=feedbackText.replace('"',' ');
+        feedback.setFeedbackText(feedbackText);
+        feedbackStore.persistFeedback(feedback);
+    }
+
+    @GET
+    @RolesAllowed({"ADMIN", "USER"})
+    @Path("/{id}/getFeedbackText")
+    public List<FeedbackDto> getTextFeedback(@PathParam("id") Long id) {
+        List<Feedback> feedbacks = feedbackStore.getFeedbackForEvent(eventStore.getEventById(id).get());
```

Comment:
What will happen if an event was not found?

Reply:
There will be an error. Optional could be solution.